ci: update WF version & triggers

vicente-astorga opened the pull request.

vicente-astorga committed:
chore: define setup.cfg files

vicente-astorga committed:
chore: remove test.sh

vicente-astorga committed:
docs: adds coverage-badge & link in README

fsologureng commented on .github/workflows/pythonapp.yml:
```diff
@@ -2,18 +2,24 @@ name: Python application
+      django-lms: true
+      django-cms: false
+      platform-eol: true
+      platform-openuchile: false
```
por favor agregar a open

vicente-astorga committed:
ci: update WF version & triggers
- set eol-release on branch conditions
- update WF version & vars to pythonapp-2
- remove paths-ignore conditions
